packages/devextreme/js/localization/messages/en.json

```
@@ -237,6 +237,7 @@
         "dxPivotGrid-filterFieldArea": "Drop Filter Fields Here",
 
         "dxScheduler-appointmentAriaLabel-group": "Group: {0}",
+        "dxScheduler-appointmentAriaLabel-recurring": "Recurring Appointment",
```

```suggestion
        "dxScheduler-appointmentAriaLabel-recurring": "Recurring appointment",
```

Please replace everywhere